javac-plugin/src/main/java/org/moditect/deptective/internal/graph/impl/Tarjan.java

```diff
@@ -0,0 +1,115 @@
+/**
+ *  Copyright 2019 The ModiTect authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.moditect.deptective.internal.graph.impl;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.moditect.deptective.internal.graph.GraphUtils;
+import org.moditect.deptective.internal.graph.INode;
+
+/**
+ */
+public class Tarjan<T extends INode> {
+
+  /** - */
+  private int                     _index                       = 0;
```

Comment:
I don't want to be too hung up style, but can we get rid of the "_" for member vars and also that whitespace?